pgsql/pc_editor.c

```diff
@@ -13,6 +13,11 @@
 #include "pc_pgsql.h"	   /* Common PgSQL support for our type */
 
 Datum pcpatch_rotate_quaternion(PG_FUNCTION_ARGS);
+Datum pcpatch_translate(PG_FUNCTION_ARGS);
+Datum pcpatch_affine(PG_FUNCTION_ARGS);
+Datum pcpoint_rotate_quaternion(PG_FUNCTION_ARGS);
+Datum pcpoint_translate(PG_FUNCTION_ARGS);
+Datum pcpoint_affine(PG_FUNCTION_ARGS);
```

Comment:
Oops, I guess I need to create a PR for those missing declarations. At the same time, I wonder why those declarations are needed.